assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -43,6 +61,21 @@ export default function SettingsView() {
 
 	return (
 		<div className="googlesitekit-setup-module googlesitekit-setup-module--analytics">
+			{ ga4Enabled && ! ga4ModuleConnected && (
+				<SettingsNotice
+					type={ TYPE_WARNING }
+					LearnMore={ () => (
+						<Link href={ documentationURL } external>
+							{ __( 'Learn more', 'google-site-kit' ) }
+						</Link>
+					) }
+					notice={ __(
+						'Your current Universal Analytics will stop collecting data on July 1, 2023',
```

Comment:
As above, if @marrrmarrr confirms we may update the copy here:

```suggestion
						'Your current Universal Analytics property will stop collecting data on July 1, 2023',
```

Reply:
As above, we can make this change...

Reply:
Updated.